src/app.rb

```diff
@@ -86,7 +117,23 @@ def list_all_authors
   end
 
   def add_book
-    puts 'Add book'
+    puts 'Please enter the publisher name?'
+    publisher = gets.chomp
+
+    puts 'Please enter the cover state of the book?'
+    cover_state = gets.chomp
+
+    archived = y_n { 'is it archived?:[Y or N]' }
+
+    puts 'Please enter publish date?'
+    publish_date = gets.chomp
+
+    puts 'Select a label for the book from the following list (not id)'
+    @label.each_with_index { |label, index| puts "[#{index}] #{label.color}" }
```

Comment:
Hi @richardoppiyo I think it would be more useful to the user to see the title of the label instead of the color.
So for example when adding a book you can select Science Fiction instead of red. Don't you think

Reply:
Yes yes sure, Thanks. I will correct that